torch_tvm/custom_tvm_ops/topi/custom_topi_ops.cc

```diff
@@ -0,0 +1,205 @@
+#include <topi/generic/injective.h>
+#include <topi/x86/default.h>
+#include <tvm/build_module.h>
+#include <tvm/relay/attrs/transform.h>
+#include <tvm/relay/expr.h>
+#include <tvm/relay/op_attr_types.h>
+#include <tvm/runtime/packed_func.h>
+
+#include <custom_tvm_ops/relay/custom_layer_norm_attrs.h>
+#include <custom_tvm_ops/relay/quantize_attrs.h>
+
+#include "custom_layer_norm.h"
+#include "custom_layer_norm_generic_sched.h"
+#include "quantize.h"
+#include "contrib/quantize.h"
+#include "generic/quantize_generic_sched.h"
+
+
+namespace tvm {
+
+using tvm::relay::OpPatternKind;
+
+class CustomTOPIOpRegisterer {
+ public:
+  CustomTOPIOpRegisterer() {
+    auto reg_ptr = runtime::Registry::Get("relay.op._Register");
+    CHECK(reg_ptr) << "Cannot find function for relay.op._Register";
+    (*reg_ptr)(
+        "nn.custom_layer_norm",
+        "TOpPattern",
+        static_cast<int>(OpPatternKind::kOutEWiseFusable),
+        10);
+    (*reg_ptr)(
+        "nn.custom_layer_norm",
+        "FTVMCompute",
+        tvm::relay::FTVMCompute(
+            [](const tvm::Attrs& attrs,
+               const tvm::Array<tvm::Tensor>& inputs,
+               const tvm::relay::Type& out_type,
+               const tvm::Target& target) -> tvm::Array<tvm::Tensor> {
+              const relay::CustomLayerNormAttrs* param =
+                  attrs.as<relay::CustomLayerNormAttrs>();
+              auto num_axis_to_normalize = param->num_axis_to_normalize;
+              auto affine = param->affine;
+              auto eps = param->eps;
+              return tvm::Array<tvm::Tensor>{topi::custom_layer_norm(
+                  inputs[0],
+                  inputs[1],
+                  inputs[2],
+                  num_axis_to_normalize,
+                  affine,
+                  eps)};
+            }),
+        10);
+    (*reg_ptr)(
+        "nn.custom_layer_norm",
+        "FTVMSchedule",
+        tvm::relay::FTVMSchedule(
+            [](const tvm::Attrs& attrs,
+               const tvm::Array<tvm::Tensor>& outs,
+               const tvm::Target& target) -> tvm::Schedule {
+              // dispatches x86 schedule when the target is "llvm" and the
+              return topi::generic::schedule_custom_layer_norm(outs);
+            }),
+        10);
+    (*reg_ptr)(
+        "nn.quantize_findminmax",
+        "TOpPattern",
+        static_cast<int>(OpPatternKind::kOutEWiseFusable),
+        10);
+    (*reg_ptr)(
+        "nn.quantize_findminmax",
+        "FTVMCompute",
+        tvm::relay::FTVMCompute(
+            [](const tvm::Attrs& attrs,
+               const tvm::Array<tvm::Tensor>& inputs,
+               const tvm::relay::Type& out_type,
+               const tvm::Target& target) -> tvm::Array<tvm::Tensor> {
+              return tvm::Array<tvm::Tensor>{topi::contrib::quantize_findminmax(inputs[0])};
```

Comment:
No need for `tvm::Array<tvm::Tensor>{`? `topi::contrib::quantize_findminmax` already returns array of Tensors?